pkg/didcomm/protocol/presentproof/states.go

```diff
@@ -202,28 +181,25 @@ func (s *requestSent) CanTransitionTo(st state) bool {
 		st.Name() == stateNameAbandoning
 }
 
-func (s *requestSent) ExecuteInbound(md *metaData) (state, stateAction, error) {
-	if md.request == nil {
-		return nil, nil, errors.New("request was not provided")
+func forwardInitial(md *metaData) stateAction {
+	return func(messenger service.Messenger) error {
+		return messenger.Send(md.Msg, md.MyDID, md.TheirDID)
 	}
+}
 
-	// creates the state's action
-	action := func(messenger service.Messenger) error {
-		// sets message type
-		md.request.Type = RequestPresentationMsgType
-		return messenger.ReplyTo(md.Msg.ID(), service.NewDIDCommMsgMap(md.request))
+func (s *requestSent) Execute(md *metaData) (state, stateAction, error) {
+	if !md.inbound {
```

Comment:
curious: I see we're still using directionality here. Did you consider using role names instead?

Reply:
related: is there a followup plan to also merge the two Handle* service functions?

Reply:
I did not think about it. `Handle*` functions have a common `service` interface. I can try to get rid of `inbound`.

Reply:
@llorllale I've merged Handle* functions and also removed `inbound` flag.